src/latticpp/ckks/params.h

```diff
@@ -29,13 +29,21 @@ namespace latticpp {
         // PN15QP827pq is the index in DefaultParams for logQP = 827 (post quantum)
         PN15QP827pq,
         // PN16QP1654pq is the index in DefaultParams for logQP = 1654 (post quantum)
-        PN16QP1654pq
+        PN16QP1654pq,
+        // Bootstrapping parameters index 0
+        BootstrapParams0,
+        // Bootstrapping parameters index 1
+        BootstrapParams1,
+        // Bootstrapping parameters index 2
+        BootstrapParams2,
+        // Bootstrapping parameters index 3
+        BootstrapParams3
```

Comment:
Nit: Why there are only 4 `BootstrapParams`? Will we add BootstrapParams4?

Reply:
These enums match corresponding default parameter sets in Lattigo, so these would only change if I change the version of Lattigo on which we depend.